standards/out-of-hours.md

```diff
@@ -0,0 +1,62 @@
+---
+category: Operating services
+---
+# Supporting services out of hours
+
+Many of our services are used outside normal office hours (9am-6pm, Monday-Friday). To make sure people are able to keep using them with confidence, even when problems arise, it's sometimes necessary to have people on call out of hours to resolve these problems.
+
+Some services don't need out of hours support. This might be because their users don't rely on it for critical parts of their job, because the users themselves don't need it outside normal office hours, or because the service is robust enough to recover from a number of classes of failure itself.
+
+There are also cases where services may need out of hours support, but we're not currently able to provide it without risking the health and well-being of our staff, usually because we don't have enough staff who are able to support it. In these cases, we should either engage a wider group of staff to support a set of services, or seek the help of suppliers to provide sustainable support.
+
+## Determining if your service needs support out of hours
+
+Not all services need out of hours support. When considering the need for out of hours support for your service, you first need to understand the risk if the service were to go down when no-one is around to fix it.
+
+The kinds of risk that might suggest out of hours support is necessary include:
+
+- Risk to someone's life
+- The MOJ's ability to provide critical, time-sensitive justice services
+- Impact to staff's and suppliers' ability to do carry out their job safely and to the best of their ability
+- Impairing citizens', suppliers', and partners' ability to interact with MOJ services in a way that meets their needs
+
+These kinds of risks should be considered alongside your service's robustness.
+
+For example, if your service can recover from most classes of issue itself, and downtime means minor impairment to a small group of people whose task is not time-sensitive, out of hours support may not be necessary. If, however, a problem with your service (whether outages, security breaches, or something else) out of hours presents a genuine risk to someone's life, it _may_ be appropriate to provide out of hours support around the clock, depending on the robustness of the system.
```

Comment:
Well, it's not necessary as far as this standard is concerned, but it might be necessary for other reasons, hence “may”. I'm happy to remove it if you think it's confusing as-is, though?

Reply:
Ah I see. The 'may' made me think it was suggesting an exception to the rule. It's not a huge thing, I was just tempted to word it a bit more strongly (I'm assuming we can always change the guideline if there are other considerations we've overlooked)